css/card.css

```diff
@@ -90,6 +90,7 @@ tcg-card {
   margin: 0.5em 0;
   text-align: center;
   font-size: 1em;
+  height:20px;
```

Comment:
why is a hardcoded height needed for the pull announcement?

Reply:
skycompass commented that the height of the section kept jumping around. This specifically was to address the 'jumping' height after initial pull, as when the pull announcement is not populated it only has margin/padding and thereafter changes in height once text is pushed in.

Reply:
can you comment in code explaining that the height is so the section doesn't jump when text first appears?

Reply:
Thanks, good idea!